packages/vnf-wallet-sdk-nodejs/test/usecases/SubmissionUseCase.test.ts

```diff
@@ -44,51 +43,48 @@ describe('PresentationSubmission Tests', () => {
         ),
         []
     );
-    const expectedExchange = (
-        exchangeJsonObj: Dictionary<any>
-    ): VCLExchange => {
-        return new VCLExchange(
-            exchangeJsonObj[VCLExchange.KeyId],
-            exchangeJsonObj[VCLExchange.KeyType],
-            exchangeJsonObj[VCLExchange.KeyDisclosureComplete],
-            exchangeJsonObj[VCLExchange.KeyExchangeComplete]
+    const expectedSubmissionResult = generatePresentationSubmissionResult(
+        PresentationSubmissionMocks.PresentationSubmissionResultJson,
+        presentationSubmission.jti,
+        presentationSubmission.submissionId
+    );
+
+    test('testSubmitPresentationSuccess', async () => {
+        const presentationSubmissionResult = await subject.submit(
+            presentationSubmission
+        );
+
+        expect(presentationSubmissionResult?.sessionToken.value).toBe(
+            expectedSubmissionResult.sessionToken.value
         );
-    };
-    const generatePresentationSubmissionResult = (
-        jsonObj: Dictionary<any>,
-        jti: string,
-        submissionId: string
-    ): VCLSubmissionResult => {
-        const exchangeJsonObj = jsonObj[VCLSubmissionResult.KeyExchange];
-        return new VCLSubmissionResult(
-            new VCLToken(jsonObj[VCLSubmissionResult.KeyToken]),
-            expectedExchange(exchangeJsonObj),
-            jti,
-            submissionId
+        expect(presentationSubmissionResult?.exchange.id).toBe(
+            expectedSubmissionResult.exchange.id
+        );
+        expect(presentationSubmissionResult?.jti).toBe(
+            expectedSubmissionResult.jti
         );
-    };
-    const expectedPresentationSubmissionResult =
-        generatePresentationSubmissionResult(
-            PresentationSubmissionMocks.PresentationSubmissionResultJson,
-            presentationSubmission.jti,
-            presentationSubmission.submissionId
+        expect(presentationSubmissionResult?.submissionId).toBe(
+            expectedSubmissionResult.submissionId
         );
-    test('testGetPresentationSubmissionSuccess', async () => {
+    });
+
+    test('testSubmitPresentationTypeFeedSuccess', async () => {
```

Comment:
I would expect a number of use case variants in these tests
1. regular submit presentation (no auth header)
2. feed submit presentation (with an auth header that returns 200)
3. Auth error when doing a feed submit presentation (with an auth header that returns 401)

Also there should be tests for the updates to presentation request
1. Regular presentation request
2. Presentation request contains the feed flag